CHANGELOG.md

```diff
@@ -176,6 +176,13 @@ is based on [Keep a Changelog](https://keepachangelog.com).
   assume that the parsed URI for `tcp://127.0.0.1:8080` returns an IP address
   when calling `authority().host`.
 
+## [0.17.7] - Unreleased
+
+### Fixed
+
+- Datagram servants are not added to their broker on creation and should no
+  longer prevent system shutdown if they never enconutered a new endpoint. 
```

Comment:
I find this confusing to read. What was the bug and how is it fixed now? Maybe something like:

```suggestion
- Datagram servants of (UDP socket managers) were not added as children to their parent broker on creation, which prevented proper system shutdown in some cases. Adding all servants consistently to the broker should make sure UDP brokers terminate correctly (#118).
```